src/ImageSharp/Formats/Png/PngOptimizeMethod.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+
+namespace SixLabors.ImageSharp.Formats.Png
+{
+    /// <summary>
+    /// Provides enumeration of available PNG optimization methods.
+    /// </summary>
+    [Flags]
+    public enum PngOptimizeMethod
+    {
+        /// <summary>
+        /// With the None filter, the scanline is transmitted unmodified.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Suppress the physical dimension information chunk.
+        /// </summary>
+        SuppressPhysicalChunk = 1,
+
+        /// <summary>
+        /// Suppress the gamma information chunk.
+        /// </summary>
+        SuppressGammaChunk = 2,
+
+        /// <summary>
+        /// Suppress the eXIf chunk.
+        /// </summary>
+        SuppressExifChunk = 4,
+
+        /// <summary>
+        /// Suppress the tTXt, iTXt or zTXt chunk.
+        /// </summary>
+        SuppressTextChunks = 8,
+
+        /// <summary>
+        /// Make funlly transparent pixels black.
+        /// </summary>
+        MakeTransparentBlack = 16,
```

Comment:
This one is really different from the other ones, I would rather place this one into a separate property inside `PngEncoder`.

Reply:
Agreed. It doesn't seem to fit. Is the naming accurate also? It's not black, it's empty.

Reply:
i think `MakeTransparentBlack` is clear in what it does, but im open for other suggestions